pages/docs/tutorials/getting-started/coming-from-openapi.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Coming from OpenAPI and beyond"
+weight: 20
+---
+
+If you're coming from OpenAPI, you must know that AsyncAPI [started as an adaptation of the OpenAPI specification](https://medium.com/asyncapi/whats-new-on-asyncapi-lots-2d9019a1869d). We wanted to have as much compatibility as possible between the two so users could reuse parts in both.
+
+Prior AsyncAPI `3.0.0` you could find lots of similarities between OpenAPI and AsyncAPI. Just bear in mind that in the world of event-driven architectures, you have more than one protocol and therefore some things are different. Check out the following comparison chart, inspired by [Darrel Miller's blog post](https://www.openapis.org/news/blogs/2016/10/tdc-structural-improvements-explaining-30-spec-part-2):
+
+import OpenAPIComparison from '../../../../components/OpenAPIComparison'
+
+<OpenAPIComparison className="my-8" />
+
+Starting with AsyncAPI `3.0.0` specification change more and is less similar to OpenAPI to enable more flexibility. You can notice the biggest change if the channel structure, where operation is separated from it. It is like you would detach operation from paths in OpenAPI. 
+
+import OpenAPIComparisonV3 from '../../../../components/OpenAPIComparisonV3'
+
+<OpenAPIComparisonV3 className="my-8" />
+
+Aside from structural differences you must know that:
+
+1. AsyncAPI is compatible with OpenAPI schemas.
+1. Message payload in AsyncAPI can be any value, not just an AsyncAPI/OpenAPI schema. For instance, it could be an Avro schema.
```

Comment:
```suggestion
1. The message payload in AsyncAPI can be any value, not just an AsyncAPI/OpenAPI schema. For instance, it could be an Avro schema.
```